app/components/containerInspect/containerInspectController.js

```diff
@@ -15,6 +15,8 @@ function ($scope, $transition$, Notifications, ContainerService) {
       $('#loadingViewSpinner').hide();
     });
   }
-  
+
+  $scope.containerInfo = { object: {} };
+  $scope.state = { TextView: false };
```

Comment:
Please relocate the $scope.state declaration on top of the `initView` function declaration, see my comment above.

Also, could you rename the `TextView` property to something more meaningful? Something like `$scope.state.DisplayTextView`.

Reply:
OK.